parsec/core/gui/main_window.py

```diff
@@ -483,6 +483,17 @@ def switch_to_tab(self, idx):
         if not ParsecApp.has_active_modal():
             self.tab_center.setCurrentIndex(idx)
 
+    def switch_to_login_tab(self):
```

Comment:
would be good to have an optional parameter to be able to provide the available device that sould be pre-selected and an `start_arg` parameter
This way clicking on a link while not connected would redirect to the connection screen then do the expected behavior once the user has typed his password ;-)

(this maybe a too big change for this PR btw ^^)

Reply:
I opened an new issue for this #1405